### ISSUE TITLE
Vote Subscription fails randomly

### ISSUE THREAD

**Issue by slemke** [th-koeln/open-product-evaluation-server#204]
For some reason the vote subscriptions fails randomly and returns a "Not authorized" even though nothing changed.

**Comment by slemke** [comment]
Now it doesn't even fail randomly. Some backend change now results in the following message:

```js
{
  "error": {
    "message": "Cannot use GraphQLSchema \"[object GraphQLSchema]\" from another module or realm.\n\nEnsure that there is only one instance of \"graphql\" in the node_modules\ndirectory. If different versions of \"graphql\" are the dependencies of other\nrelied on modules, use \"resolutions\" to ensure only one version is installed.\n\nhttps://yarnpkg.com/en/docs/selective-version-resolutions\n\nDuplicate \"graphql\" modules cannot be used at the same time since different\nversions may have different capabilities and behavior. The data from one\nversion used in the function from another could produce confusing and\nspurious results."
  }
}
```
And yes there is only on instance of `graphql` in `/node_modules`.

**Comment by slemke** [comment]
fixed  in #226
